apps/wing-console/console/server/src/utils/compiler.ts

```diff
@@ -44,6 +44,7 @@ export const createCompiler = ({
       await events.emit("compiling");
       const simfile = await wing.compile(wingfile, {
         platform,
+        testing: true,
       });
```

Comment:
If the app is compiled without `testing: true`, then the console doesn't see any tests...